corehq/apps/settings/views.py

```diff
@@ -714,7 +717,8 @@ def get_create_item_data(self, create_form):
                 "domain": new_api_key.domain or _('All Projects'),
                 'ip_allowlist': new_api_key.ip_allowlist,
                 'created': new_api_key.created.isoformat(),
-                'is_active': new_api_key.is_active,
+                "deactivated_on": None,
+                'is_active': None,
```

Comment:
nit: why `None`, can't we say `True`?-- it feels confusing to me

Reply:
I agree - I couldn't remember why I did it this way, but I just tried it out.  The reason I had done this at first was to show it as a blank cell in the table, but I see now that that's still overly complicated (I could just omit the field).  Still, looking at this and the other fields, I see the timestamp uses a different display format when created and when displayed in the big table at the top, and the `ip_allowlist` doesn't fallback to the string "All IP Addresses" here either.  I'm switching this to re-use the `_to_json` method, which should make this make more sense.